config/config.go

```diff
@@ -17,6 +17,7 @@ import (
 	"gopkg.in/yaml.v2"
 
 	"github.com/canonical/candid/idp"
+	"github.com/canonical/candid/internal/mfa"
```

Comment:
I think it might be better if you passed a config into the mfa package rather than create an mfa Authenticator in this package. unless that causes an unexpected problem.

Reply:
done